concent_api/core/payments/backends/sci_backend.py

```diff
@@ -100,13 +101,14 @@ def force_subtask_payment(
 ) -> str:
     assert isinstance(requestor_eth_address, str) and len(requestor_eth_address) == ETHEREUM_ADDRESS_LENGTH
     assert isinstance(provider_eth_address, str) and len(provider_eth_address) == ETHEREUM_ADDRESS_LENGTH
-    assert isinstance(value, int) and value > 0
     assert isinstance(subtask_id, str)
 
+    validate_value_is_int_convertible_and_non_negative(value)
```

Comment:
Uncle Bob says "Aggghhhhrrrrr" ;). I know that this function was here before, but if You can please rename it to `validate_value_correctness` and in this function:
```python
def validate_value_correctness(value: int) -> None:
    validate_value_is_int_convertible(value)
    validate_positive_integer_value(value)
```
Now `validate_value_is_int_convertible_and_non_negative` mixes abstractions levels

Reply:
"`validate_value_correctness`", seriously? What about that functions should have names that specifically says what they do?
I'm fine with refactoring that function, to split it in to two separate.

Reply:
Done